### ISSUE TITLE
Newest update [5.1.20] completely breaks Gambling.

### ISSUE THREAD

**Issue by CoryWithNoE** [D3lta/unraid-container-nadekobot#1]
I updated the bot to the newest 5.1.20 version, which completely broke the gambling mechanic on my server.

Any time anyone ran .timely, they received an error saying "Error: dkey timely_button not found!"

My first troubleshooting step was deleting the nadeko appdata folder, I then redownloaded the latest version which caused the same issue. I then removed all the files again, but this time used 5.1.15 and it's working flawlessly again.

**Comment by CoryWithNoE** [comment]
The only message I received in my logs when running the command was this. 

----

[21:49:20 WRN] | #0 | 'timely_button' key is missing from 'en-US' response strings. You may ignore this message.

**Comment by D3lta** [comment]
Looks like new response strings were added, but because the data folder is only copied once and not modified if existing data is found it does not not copy over any newly added strings from the newer data folder.

A current workaround would be to manually copy over the new strings, by entering the console
```bash
# mv /data/bot/strings /data/bot/strings.bak
# cp -R /tmp/data/strings/ /data/bot/strings
```
and then restart the container

**Comment by D3lta** [comment]
Technically I would not want to touch any of those files as they could be modified by the user.

Would you like an option that it automatically copies over new strings into your data folder?

**Comment by D3lta** [comment]
Fixed in #2 
On each startup it will copy the strings, live on version current latest version (v5.2.4)
